operations/addresses.md

```diff
@@ -0,0 +1,231 @@
+
+#### Address items
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the [Address](configuration.md#address). |
+| `AccountId` | string | required | Unique identifier of a [Company](companies.md#company) or a [Customer](customers.md#customer) within the enterprise. |
+| `AccountType` | string | required | A discriminator specifying the [type of account](#account-type), e.g. customer or company. |
+| `Line1` | string | optional | First line of the address. |
+| `Line2` | string | optional | Second line of the address. |
+| `City` | string | optional | The city. |
+| `PostalCode` | string | optional | Postal code. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](countries.md#country). |
+| `CountrySubdivisionCode` | string | optional | ISO 3166-2 code of the administrative division, e.g. `DE-BW`. |
+| `Latitude` | number | optional | The latitude. |
+| `Longitude` | number | optional | The longitude. |
+
+#### Account type
+
+* [Company](companies.md#company)
+* [Customer](customers.md#customer)
+
+## Add addresses
+
+Adds one or more new addresses to the system and assign them to specified accounts.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/addresses/add`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "Addresses": [
+        {
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `Addresses` | array of [Address items](#address-items), max 1000 items | required | Collection of addresses to be created. |
+
+### Response
+
+```javascript
+{
+    "Addresses": [
+        {
+            "Id": "fc7b2df3-de66-48a6-907d-af4600ecd892",
+            "AccountId": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "AccountType": "Customer",
+            "Line1": "Rheinlanddamm 207-209",
+            "Line2": null,
+            "City": "Dortmund",
+            "PostalCode": "44137",
+            "CountryCode": "DE",
+            "CountrySubdivisionCode": null,
+            "Latitude": null,
+            "Longitude": null
+        }
+    ],
+    "Cursor": null
```

Comment:
`AddressResult` doesn't have a cursor as it doesn't make sense.

Reply:
![image](https://user-images.githubusercontent.com/24482879/201925183-7af40710-c2d2-4cf6-8420-d13a2ca0f527.png)

Reply:
It does. And postman will return it.

Reply:
Based on a call with you, I'll remove those (Vojta's PR had those too, I'll fix it there as well)